app/namespaces/airtable_scraper/airtable_scraper_controller.py

```diff
@@ -49,7 +49,16 @@ def get(self):
             # Get all records from using airtable API
             records, status_code = get_all_records(visualize_on_serotracker_filter, airtable_fields_json_name)
             file_created_datetime = datetime.now()
-        result = {"airtable_request_status_code": status_code, "records": records, "updated_at": file_created_datetime}
+
+        max_sampling_end_date = datetime.min
```

Comment:
This variable should be max_pub_date throughout - we care about the publication date here, not the study end date!
The field you're accessing reflects that, but the variable name should also

Reply:
lol whoops forgot to update that